bit9PlatformAPI/examples/connectors/Metascan.py

```diff
@@ -0,0 +1,247 @@
+the original script. Hopefully this will indemnify both Bit9 and me from any issues encountered by you if you use this
+script.
+
+If you can find a better way to work any of this, go for it. I'd ask that you update the script in GitHub if you do though.
+Thanks and enjoy!
+
+-Patrick
+
+Disclaimer
++++++++++++++++++++
+By accessing and/or using the samples scripts provided on this site (the "Scripts"), you hereby agree to the following terms:
+The Scripts are exemplars provided for purposes of illustration only and are not intended to represent specific
+recommendations or solutions for API integration activities as use cases can vary widely.
+THE SCRIPTS ARE PROVIDED "AS IS" WITHOUT WARRANTIES OF ANY KIND, EITHER EXPRESS OR IMPLIED.  BIT9 MAKES NO REPRESENTATION
+OR OTHER AFFIRMATION OF FACT, INCLUDING BUT NOT LIMITED TO STATEMENTS REGARDING THE SCRIPTS' SUITABILITY FOR USE OR PERFORMANCE.
+IN NO EVENT SHALL BIT9 BE LIABLE FOR SPECIAL, INCIDENTAL, CONSEQUENTIAL, EXEMPLARY OR OTHER INDIRECT DAMAGES OR FOR DIRECT
+DAMAGES ARISING OUT OF OR RESULTING FROM YOUR ACCESS OR USE OF THE SCRIPTS, EVEN IF BIT9 IS ADVISED OF OR AWARE OF THE
+POSSIBILITY OF SUCH DAMAGES.
+
+Requirements
++++++++++++++++++++
+
+- Bit9 API client (included) which requires requests Python module
+
+- Metascan API Key
+
+- Bit9 Platform Server 7.2.1 or later
+- Bit9 API Token (generated in Bit9 Console)
+- Bit9 RBAC permission enabled for 'Extend connectors through API'
+
+Required python modules can be installed using tools such as easy_install or pip, e.g.
+    easy_install requests
+
+Configuring Connector
++++++++++++++++++++++++
+
+Please update the script with appropriate vt_token, Bit9 server address and Bit9 token at the bottom of the script.
+
+
+Start the script. No parameters are required. It will process analysis requests from teh Bit9 Platform as long as it is running.
+Script execution can be terminated with ctrl+c.
+"""
+
+import requests
+import datetime
+import sys
+import bit9api
+
+# -------------------------------------------------------------------------------------------------
+# MS connector class. Initialization where keys are specified is done at the bottom of the script
+class metascanConnector(object):
+    def __init__(self, bit9, ms_token=None, download_location=None):
+        """ Requires:
+                server -    URL to the Bit9Platform server.  Usually the same as
+                            the web GUI.
+                sslVerify - verify server SSL certificate
+                token - this is for CLI API interface
+        """
+
+        if ms_token is None:
+            raise TypeError("Missing required MetaScan authentication token.")
+        #My test system does not require a API token so I'm not adding it to the JSON call
+        self.ms_token = ms_token
+        self.ms_url = 'http://yourMetaScanServer/metascan_rest/'
```

Comment:
Please add the metascan server hostname to the `__init__()` parameters for your class